src/services/export.js

```diff
@@ -224,24 +236,32 @@ const saveEpub = async page => {
   try {
     // todo: parse title in appropriate language
     introduction.title = 'Introduction';
-    introduction.data = await page.$eval(
-      '.ils-description',
-      el => el.innerHTML
-    );
+    introduction.data = await page.$eval('.description p', el => el.innerHTML);
   } catch (err) {
     console.error(err);
   }
 
   // get body for epub
-  const body = await page.$$eval('div.tab-pane', phases =>
+  // use the export class to differentiate from tools content
+  const body = await page.$$eval('.export > section', phases =>
     phases.map(phase => ({
-      title: phase.getAttribute('phase-title'),
-      data: phase.innerHTML,
+      title: phase.getElementsByClassName('name')[0].innerHTML,
+      data: phase.getElementsByClassName('resources')[0].innerHTML,
     }))
   );
 
+  // get tools for epub
+  const tools = {};
+  try {
+    // todo: parse title in appropriate language
+    tools.title = 'Tools';
+    tools.data = await page.$eval('.tools > section', el => el.innerHTML);
+  } catch (err) {
+    console.error(err);
```

Comment:
@pyphilia, let's start using `Logger.error(err)` for the errors, instead of `console`.